### PR TITLE
feat: login and signup page

### PR DESCRIPTION
**Pull request by pankaj-prog**
login page: 
BlitheBrains - Login

sign up page: 
BlitheBrains - Signup

### CODE REVIEW

File: src/pages/auth/auth.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  gap: 0.5rem;
+}
+
+.auth-form .input-container i {
+  color: var(--GRAY);
+}
+
+.auth-form .input-container input {
+  font-size: 1rem;
+  min-width: 30ch;
```
Comment (bhtibrewal) [inline src/pages/auth/auth.css:61]: ch?

**Review by bhtibrewal — APPROVED** [review]
Appreciate: the UI looks really clean, you are good to merge

File: src/pages/auth/auth.css (hunk 1 above)
Comment (pankaj-prog) [reply src/pages/auth/auth.css:61]: ch means characters here

Comment (gauravsinhaweb) [reply src/pages/auth/auth.css:61]: > ch?

word character

File: src/pages/auth/auth.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  outline: none;
```
Comment (gauravsinhaweb) [inline src/pages/auth/auth.css]: you shouldn't give outline none, for accessibility

**Review by gauravsinhaweb — APPROVED** [review]
everything looks clean, good to merge.

**Pull request merged**